dev/docker/doris/start.sh

```diff
@@ -65,13 +68,56 @@ if [ "$fe_started" = false ]; then
   exit 1
 fi
 
+# check for be started
+be_started=false
+for i in {1..10}; do
+  sleep 5
+  # check localhost:8040/api/health return contains content '"status": "OK"' to see if the be is ready
+  url="localhost:8040/api/health"
+
+  echo "check be for the $i times"
+  response=$(curl --silent --request GET $url)
+
+  if echo "$response" | grep -q "\"status\": \"OK\""; then
+    be_started=true
+    echo "Doris be started successfully, response: $response"
+    break
+  else
+    echo "check failed, response: $response"
+  fi
+done
+
+if [ "$be_started" = false ]; then
+  echo "Doris be failed to start"
+
+  cat ${DORIS_BE_HOME}/log/*
```

Comment:
By doing `cat ${DORIS_BE_HOME}/log/*`, we can review the log of the backend server?

Reply:
yes, we can cat all backend log to stdout, so Gravitino could print to log file